File: java/EBook.java
public class EBook extends Book {
    private double fileSize;

    public EBook(String title, String author, String ISBN, double price, int stock, double fileSize) {
        super(title, author, ISBN, price, stock);
        this.fileSize = fileSize;
    }

    public EBook(String title, String author, String ISBN, double price, int stock, double fileSize) {
        super(title, author, ISBN, price, stock);
        this.fileSize = fileSize;
    }

    @Override
    public void displayDetails() {
        super.displayDetails();
        System.out.println("FileSize: " + fileSize);
    }
}

File: python/26/2/1/EBook.py
from Book import Book
class EBook(Book):
    def __init__(self, title, author, ISBN, price, stock,fileSize):
        super().__init__(title, author, ISBN, price, stock)
        self.__fileSize=fileSize
    def displayDetails(self):
        super().displayDetails()
        print("FileSize: ",self.__fileSize)
